systemdfiles/mozdefweb.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=MozDef Web Interface (meteor)
+After=mongod.service
+
+[Service]
+# Requires systemd version 211 or newer
+User=mozdef
+Group=mozdef
+# Client Env Vars
+Environment=HOME="/opt/mozdef"
+Environment=MONGO_URL=mongodb://localhost:3002/meteor
+Environment=ROOT_URL=http://your.domain.name.here
+Environment=PORT=3000
+Environment=NODE_TLS_REJECT_UNAUTHORIZED="0"
```

Comment:
Yes, this disables it, because node doesn't recognize our CA.
This will be resolved soon as node was working on allowing users to point to their ca of choice.
This was fixed using a different method in the new release. This PR will be closed without merging.